src/plugins/mix.ts

```diff
@@ -1,18 +1,34 @@
 import { AnyColor } from "../types";
 import { Plugin } from "../extend";
 import { mix } from "../manipulate/mix";
+import { Colord } from "../colord";
 
 declare module "../colord" {
   interface Colord {
     /**
      * Produces a mixture of two colors through CIE LAB color space and returns a new Colord instance.
      */
     mix(color2: AnyColor | Colord, ratio?: number): Colord;
+
+    /**
+     * Generates a tints palette based on original color.
+     */
+    tints(colors: number): Colord[];
+
+    /**
+     * Generates a shades palette based on original color.
+     */
+    shades(colors: number): Colord[];
+
+    /**
+     * Generates a tones palette based on original color.
+     */
+    tones(colors: number): Colord[];
```

Comment:
Since it's optional.

```suggestion
    /**
     * Generates a tints palette based on original color.
     */
    tints(colors?: number): Colord[];

    /**
     * Generates a shades palette based on original color.
     */
    shades(colors?: number): Colord[];

    /**
     * Generates a tones palette based on original color.
     */
    tones(colors?: number): Colord[];
```